commercetools-models/src/main/java/io/sphere/sdk/extensions/Extension.java

```diff
@@ -0,0 +1,71 @@
+package io.sphere.sdk.extensions;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import io.sphere.sdk.annotations.*;
+import io.sphere.sdk.models.Reference;
+import io.sphere.sdk.models.Resource;
+import io.sphere.sdk.models.WithKey;
+
+import javax.annotation.Nullable;
+import java.util.List;
+
+
+@JsonDeserialize(as=ExtensionImpl.class)
+@ResourceValue
+@ResourceInfo(pluralName = "extensions", pathElement = "extensions")
+@HasByIdGetEndpoint(javadocSummary = "Retrieves a extension by a known ID.")
+@HasByKeyGetEndpoint(javadocSummary = "Retrieves a extension by a known Key.")
+@HasQueryModel(baseInterfaces = {"io.sphere.sdk.queries.QueryModel<io.sphere.sdk.extensions.Extension>"})
+@HasQueryEndpoint
+@HasCreateCommand(javadocSummary = "Creates an extension")
+@HasUpdateCommand(javadocSummary = "Updates an extension.", updateWith = {"key","id"})
+@HasDeleteCommand(javadocSummary = "Deletes an extension.", deleteWith = {"key","id"})
+public interface Extension extends Resource<Extension>, WithKey {
+
+
+    @IgnoreInQueryModel
+    @HasUpdateAction
+    List<Trigger> getTriggers();
+
+    @IgnoreInQueryModel
+    @HasUpdateAction
+    Destination getDestination();
+
+    @Override
+    @Nullable
+    @IgnoreInQueryModel
```

Comment:
Why do we ignore this property in our query model? In our other resources we don't ignore the key.

Reply:
@katmatt  good catch 👍